src/plugins/messenger/MessengerPreview/components/MessengerListTemplate/components/MessengerListTemplateElement.tsx

```diff
@@ -45,31 +50,38 @@ export const getMessengerListTemplateElement = ({ React, styled }: MessagePlugin
         const imgStyle: React.CSSProperties = {
             backgroundImage: getBackgroundImage(image_url)
         }
+        const messengerSubtitleId = `webchatListTemplateSubtitle-${uuid.v4()}`
+        const messengerAriaLabel  = default_action ? title + " .Opens in new tab" : title;
 
         return (
-            <Root
-				role="listitem"
-                onClick={default_action && (e => onAction(e, default_action))}
-                className="webchat-list-template-element"
-                style={default_action ? { cursor: "pointer" }:{}}
-            >
-                <div>
-                    <MessengerTitle className="webchat-list-template-element-title" dangerouslySetInnerHTML={{__html: title}} />
-                    <MessengerSubtitle className="webchat-list-template-element-subtitle" dangerouslySetInnerHTML={{__html: subtitle}} config={config} />
-                    {button && (
-                        <ListButton
-                            onClick={e => {e.stopPropagation(); onAction(e, button)}}
-                            className="webchat-list-template-element-button"
-                            dangerouslySetInnerHTML={{__html: getButtonLabel(button)}}
-                        />
+            <div role="listitem">
+                <Root
+                    role={default_action ? "link" : undefined} 
+                    onClick={default_action && (e => onAction(e, default_action))}
+                    className="webchat-list-template-element"
+                    style={default_action ? { cursor: "pointer" }:{}}
+                    aria-label={messengerAriaLabel} 
+                    aria-describedby={subtitle ? messengerSubtitleId : undefined}
+                    tabIndex={default_action ? 0 : -1}
```

Comment:
this isn't keyboard accessible, either. I can focus the item via keyboard, but hitting "return" does not open the link

Reply:
Fixed